controllers/idler/idler_controller_test.go

```diff
@@ -80,9 +80,11 @@ func TestReconcile(t *testing.T) {
 		idler := &toolchainv1alpha1.Idler{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:              "being-deleted",
+				Finalizers:        []string{"toolchain.dev.openshift.com"},
 				DeletionTimestamp: &now,
 			},
 			Spec: toolchainv1alpha1.IdlerSpec{TimeoutSeconds: 30},
+			//Status: toolchainv1alpha1.IdlerStatus{Conditions: make([]toolchainv1alpha1.Condition, 0)},
```

Comment:
```suggestion
```

Reply:
I didn't have to comment out the status - uncommented in [0b5289e](https://github.com/codeready-toolchain/member-operator/pull/551/commits/0b5289e17f8337c2eeba756dd94a524b87163fd0) instead of deleting